apps/meteor/app/lib/server/lib/validateCustomMessageFields.ts

```diff
@@ -0,0 +1,37 @@
+import Ajv from 'ajv';
+import mem from 'mem';
+
+const ajv = new Ajv();
+
+const customFieldsValidate = mem((customFieldsSetting: string) => {
+	const schema = JSON.parse(customFieldsSetting);
```

Comment:
What happens if this throws an error?

Reply:
It throws the error that you can check in the unit test named `should not allow to pass invalid custom fields config`